flink-table/flink-sql-client/src/test/java/org/apache/flink/table/client/cli/CliTableauResultViewTest.java

```diff
@@ -484,7 +484,7 @@ public void testCancelStreamingResult() throws Exception {
     }
 
     @Test
-    public void testFailedStreamingResult() {
+    void testFailedStreamingResult() {
         final Configuration testConfig = new Configuration();
         testConfig.set(EXECUTION_RESULT_MODE, ResultMode.TABLEAU);
         testConfig.set(RUNTIME_MODE, RuntimeExecutionMode.STREAMING);
```

Comment:
The line started at 516 could be simplified with usage of `assertThat(terminalOutput).hasToString(`
The line started at 532 could be a bit simplified as 
```java
assertThat(mockExecutor.getNumCancelCalls()).isOne();
```